application/engine/services/context_budget_allocator.py

```
@@ -84,6 +85,13 @@ def get_final_context(self) -> str:
                 if slot.content.strip():
                     parts.append(f"\n=== {slot.name.upper()} ===\n{slot.content}")
         
+        # 追加强制收束指令
+        if self.expired_foreshadows:
+            parts.append("\n=== 🚨强制剧情收束令🚨 ===\n" + 
+                         "以下伏笔已超出预期揭晓章节，必须在本章或本节拍的行文中，通过回忆、对话、意外发展或直接揭露等方式去解答或明显推进悬念：\n" + 
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)